### PR TITLE
Add logging configuration options to documentation

### PR DESCRIPTION
**Pull request by dennyac**
Adding the queueSize and discardingThreshold logging configuration options to the documentation. This is related to https://github.com/dropwizard/dropwizard/issues/1856

### CODE REVIEW

File: docs/source/manual/configuration.rst [hunk 1]
```diff
@@ -603,6 +603,10 @@ Name                   Default      Description
 ====================== ===========  ===========
 type                   REQUIRED     The appender type. Must be ``console``.
 threshold              ALL          The lowest level of events to print to the console.
+queueSize              256          The maximum capacity of the blocking queue.
+discardingThreshold    51           By default, when the blocking queue has 20% capacity remaining,
```
Comment (evnm) [inline docs/source/manual/configuration.rst]: I had to dig into Logback source code to figure out where 51 comes from. Let's call out that if no discarding threshold is specified, then a default of `queueSize / 5` is used.

**Comment by dennyac** [comment]
The 51 comes from [here](https://github.com/qos-ch/logback/blob/master/logback-core/src/main/java/ch/qos/logback/core/AsyncAppenderBase.java#L103-L104). I'll update the documentation.

File: docs/source/manual/configuration.rst (hunk 1 above)
Comment (nickbabcock) [inline docs/source/manual/configuration.rst]: nvm, ignore this comment, I forgot `-1` meant undefined 😞

**Comment by dennyac** [comment]
I just updated the description. Let me know if it looks good or requires more changes.

**Comment by evnm** [comment]
@jplock Any reason this isn't suited for the 1.0.6 milestone?

**Comment by jplock** [comment]
@evnm I've been tagging the milestone based on the destination branch of the pull request, not necessarily on the functionality.  I wish the main pull request list included the destination branch.  :)

**Pull request merged**